examples/testplans/release.json

```diff
@@ -3,6 +3,9 @@
     "description": "Test Plan to be performed before a release is cut",
 
     "tests": [
+	{"name": "Get the latest available requirements",
+	 "description": "Update your systems requirements to the latest pip ones to avoid new deps breakages, run `$ pip install -r requirements.txt --upgrade` inside Avocado sources"},
```

Comment:
I have mixed feelings about this, because a developer may choose to have the requirements provided by other source (such as distro packages).  IMO, this is more akin to testing the deployment of Avocado, which we're lacking, but doesn't quite fit here.

Reply:
I do use RPMs for dependencies, but using virtual-env (or simply removing the `~/.local/lib/python2...`) should allow us to do this and might catch ugly issues. I don't say this is an urgent step, but honestly I find this check more important than several later ones.

Anyway this was just a suggestion so if you don't think it's important enough (or viable) I'm fine with keeping it as is.